server/portal/apps/onboarding/steps/system_creation.py

```diff
@@ -16,10 +16,10 @@ def __init__(self, user):
         super(SystemCreationStep, self).__init__(user)
 
     def display_name(self):
-        return "Creating Storage Systems"
+        return "Storage"
 
     def description(self):
-        return """	Setting up access to the storage systems. No action required."""
+        return """	Setting up access to data files on the storage systems. No action required."""
```

Comment:
**Suggestion**
The tab between `"""` and `Setting` may cause white-space in output. I suggest to delete it to avoid possible white-space and to be consistent with other descriptions.

Reply:
Latest commit should have fixed this, thanks for the suggestion!